### PR TITLE
Release 40.6.0

### PR DESCRIPTION
**Pull request by richmadetech**
This is the release PR for version 40.6.0.

Refer to [How to prepare a release](https://github.com/uktrade/data-hub-api/blob/develop/docs/How%20to%20prepare%20a%20release.md) for further information and the next steps.

### CODE REVIEW

**Review by cgsunkel — APPROVED** [review]

File: datahub/activity_stream/event/test/test_event_views.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                    },
+                },
+            ],
+        }
+
+
+@pytest.mark.django_db
+def test_null_event_region(api_client):
+    """
+    Test that we can handle event region  being None
```
Comment (vfarah-if) [inline datahub/activity_stream/event/test/test_event_views.py:70]: You can remove this as the name implies the same thing, same for others below

File: datahub/activity_stream/event/test/test_event_views.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+
+@pytest.mark.django_db
+def test_null_event_organiser(api_client):
+    """
+    Test that we can handle event Organiser  being None
+    """
+    with freeze_time() as frozen_datetime:
+        EventFactory(organiser=None)
+        frozen_datetime.tick(datetime.timedelta(seconds=1, microseconds=1))
```
Comment (vfarah-if) [inline datahub/activity_stream/event/test/test_event_views.py:87]: Extract test fixture for this arrangement to use in all the tests

File: datahub/activity_stream/event/test/test_event_views.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+@pytest.mark.django_db
+def test_null_event_lead_team(api_client):
+    """
+    Test that we can handle event Lead Team being None
+    """
+    with freeze_time() as frozen_datetime:
+        EventFactory(lead_team_id=None)
+        frozen_datetime.tick(datetime.timedelta(seconds=1, microseconds=1))
+        response = hawk.get(api_client, get_url('api-v3:activity-stream:events'))
```
Comment (vfarah-if) [inline datahub/activity_stream/event/test/test_event_views.py:101]: See above

File: datahub/activity_stream/event/test/test_event_views.py [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+@pytest.mark.django_db
+def test_null_event_location_type(api_client):
+    """
+    Test that we can handle event location Type  being None
+    """
+    with freeze_time() as frozen_datetime:
+        EventFactory(location_type_id=None)
+        frozen_datetime.tick(datetime.timedelta(seconds=1, microseconds=1))
+        response = hawk.get(api_client, get_url('api-v3:activity-stream:events'))
```
Comment (vfarah-if) [inline datahub/activity_stream/event/test/test_event_views.py:114]: ...

File: datahub/activity_stream/event/test/test_event_views.py (hunk 1 above)
Comment (richmadetech) [reply datahub/activity_stream/event/test/test_event_views.py]: Removed, and refactored

**Review by tomkingston — CHANGES_REQUESTED** [review]
This cannot include code changes or it will put master and develop out of sync - if you wish to get that code change in you need to open a new branch and move it through the whole process unfortunately :(

File: datahub/activity_stream/event/test/test_event_views.py [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+def run_none_type_tests(api_client):
+    with freeze_time() as frozen_datetime:
+        frozen_datetime.tick(datetime.timedelta(seconds=1, microseconds=1))
+        response = hawk.get(api_client, get_url('api-v3:activity-stream:events'))
+
+        assert response.status_code == status.HTTP_200_OK
+
+
+@pytest.mark.django_db
```
Comment (vfarah-if) [inline datahub/activity_stream/event/test/test_event_views.py:75]: May have added a duplicate

**Pull request closed without merging**